ui/frontend/build_src/src/components/creationPanel/advancedSettings/index.tsx

```diff
@@ -145,9 +145,9 @@ function SettingsList() {
         Generate in parallel:
         <input
           type="number"
-          value={num_outputs}
+          value={parallelCount}
           onChange={(e) =>
-            setRequestOption("num_outputs", parseInt(e.target.value, 10))
+            setParallelCount(parseInt(e.target.value, 10))
```

Comment:
in the original code these were swapped, which seemed strange, but I went with it, but for this fix, they needed to be in their correct place.